tests/test_data_transforms.py

```diff
@@ -0,0 +1,22 @@
+from src.transforms import Compose
+from src.transforms.augment import RandomScaleAnisotropic, RandomTranslate
+
+
+def test_random_translate():
+    translate = RandomTranslate(delta=[0.5, 0.5, 0.5])
+    assert translate.delta == [0.5, 0.5, 0.5]
```

Comment:
I wonder if there's a way to test that these functions are doing what they're supposed to...

Maybe giving them a data object and checking that after running the transform the position is within what it's supposed to be?